DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_SearchResultsCountWithAlert.cshtml

```diff
@@ -5,5 +5,5 @@
   @{
     var count = Model.MatchingSearchResults;
   }
-  <span class="results-count">@count matching @(count == 1 ? "result" : "results")</span>
+  <span role="alert" class="results-count">@count matching @(count == 1 ? "result" : "results")</span>

```

Comment:
Doesn't look like this file is used anywhere. Is this being kept for some reason, or can it be removed?

Reply:
If we do end up keeping this file, I'd suggest renaming the pagination partials so that we only have partials `WithAlert` rather than some `WithAlert` and others `WithoutAlert`.

Reply:
Yeah this file is needed for the Email Delegates page. Will rename all the other partials as well.